frontend/src/services/mocks/TalkEntry.ts

```diff
@@ -0,0 +1,8 @@
+export type TalkEntry = {
+  uid: string;
+  title: string;
+  abstract: string;
+  date: string;
+  invitation_link: string;
+  youtube_link: string;
```

Comment:
Do we want those two attribute to be possibly `undefined`?

I think `youtube_link` should  be `youtube_link?` so we are not forced to define a YouTube link before the upload of a video.

We may also generate the invitation link later, so I think the rest of the code should be a bit refactored to work even with `invitation_link?`.

Reply:
it's specified and the impacts achieved